acceptance-tests/upgrade/update_and_upgrade_redis_test.go

```diff
@@ -0,0 +1,87 @@
+package upgrade_test
+
+import (
+	"csbbrokerpakaws/acceptance-tests/helpers/apps"
+	"csbbrokerpakaws/acceptance-tests/helpers/brokers"
+	"csbbrokerpakaws/acceptance-tests/helpers/random"
+	"csbbrokerpakaws/acceptance-tests/helpers/services"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Redis", Label("redis"), func() {
+	When("upgrading broker version", func() {
+		It("should continue to work", func() {
+			By("pushing latest released broker version")
+			serviceBroker := brokers.Create(
+				brokers.WithPrefix("csb-aws-redis"),
+				brokers.WithSourceDir(releasedBuildDir),
+				brokers.WithReleaseEnv(),
+			)
+			defer serviceBroker.Delete()
+
+			By("creating a service instance")
+			serviceInstance := services.CreateInstance(
+				"csb-aws-redis",
+				services.WithPlan("small"),
+				services.WithBroker(serviceBroker),
+			)
+			defer serviceInstance.Delete()
+
+			By("pushing the unstarted app twice")
+			appOne := apps.Push(apps.WithApp(apps.Redis))
+			appTwo := apps.Push(apps.WithApp(apps.Redis))
+			defer apps.Delete(appOne, appTwo)
+
+			By("bindingOne the apps to the Redis service instance")
```

Comment:
*suggestion*: Maybe, "binding the apps..."?

Reply:
Rename gone wrong - good catch